tes-adapter/src/main/java/com/epam/pipeline/adaptor/controller/TesAdapterController.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.pipeline.adaptor.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TesAdapterController {
+
+    @GetMapping("/v1/tasks/service-info")
+    public String getSimpleResponse(){
```

Comment:
Please look at the class `ResponseEntity` and try to use it here

Reply:
You also can use '@responsebody' annotation

Reply:
fixed